### ISSUE TITLE
Accessibility bug via AI4D: Name of the "Image" is not correct i.e. "MainInstructionIcon" instead of "Warning Icon".

### ISSUE THREAD

**Issue by Benjin** [microsoft/vscode#79072]
Upstream via https://github.com/microsoft/azuredatastudio/issues/6666.  More details there.

TL;DR:

When producing a warning dialog box via `vscode.window.showWarningMessage()`, the accessible name of the image is "MainInstructionIcon", when it should be something like "Warning Icon" for screen-readers.

![image](https://user-images.githubusercontent.com/1609827/62980897-adedac00-bddc-11e9-86fa-d8cc698747e1.png)

**Comment by isidorn** [comment]
Please file this bug against electron. That is a native electron dialog.
Create a small electron app which triggers a native dialog with a warning a report the issue here https://github.com/electron/electron
Thanks
